docs/_data/table/common-configuration.csv

```diff
@@ -1,19 +1,20 @@
 propertyName,defaultValue
+alluxio.conf.directory,${alluxio.home}/conf
 alluxio.debug,false
 alluxio.home,/mnt/alluxio_default_home
 alluxio.logs.dir,${alluxio.home}/logs
 alluxio.keyvalue.enabled,false
 alluxio.keyvalue.partition.size.bytes.max,512MB
-alluxio.metrics.conf.file,${alluxio.home}/conf/metrics.properties
+alluxio.metrics.conf.file,${alluxio.conf.directory}/metrics.properties
```

Comment:
previously we used only `dir`, so I've switched to using `dir` for the new configuration properties